### ISSUE TITLE
Add Curios support for backpack

### ISSUE THREAD

**Issue by GeorgeOrwell1** [MysticMods/Druidcraft#96]
**Describe your suggestion:**

The Travel Pack item should be able to be put into a Curios slot (back slot, backpack slot whatever) to act as an extra slot for the Travel Pack. The Travel Pack can then later be opened with a configured key (maybe B by default?) to access its contents. It will also be rendered/displayed as if you were to have it in your inventory.

**Describe alternatives you've considered:**

An alternative solution could be to be able to wear the backpack on the vanilla chestplate slot (although that would it make you unable to set another armor piece on there unless you would be able to "pair" the so-called backpack to any chestplate...)

**Additional context:**

**Comment by crazyirishwrencher** [comment]
Another vote for this. Would be a great little QoL feature.

**Comment by Boxbott** [comment]
I feel like this is too op, the travel pack is perfectly balanced right now, it's price and all. Plus, this would really make it a generic backpack.

**Comment by GeorgeOrwell1** [comment]
> I feel like this is too op, the travel pack is perfectly balanced right now, it's price and all. Plus, this would really make it a generic backpack.

I mean, there is a reason why so many backpack mods add a backpack slot...
If you still would want the "old" behaviour, you could just... not install Curios. Even if you would implement the second option, it still wouldn't add an extra slot (if that's what you mean by "op").

**Comment by ChloeDawn** [comment]
Allowing it to be equipped in the chest-plate slot sounds like a fairly balanced alternative option

**Comment by wildcardgamez** [comment]
Agreed, it just would feel right. And it's not op because all your doing is having it somewhere other than your Hotbar, it still only has 6 slots.
